src/plugins/syscalls/syscalls.cpp

```diff
@@ -218,7 +255,9 @@ static event_response_t win_cb(drakvuf_t drakvuf, drakvuf_trap_info_t *info) {
                     printf("-");
             }
         }
-
+        if(strstr(info->trap->name,"_Return")!=NULL) {
```

Comment:
Instead of doing this by naming the trap, how about doing a separate callback that only gets called for the return trap?